api.yaml

```diff
@@ -30,11 +30,17 @@ x-runtime:
 x-transportd:
   backends:
     - app
+    - nexpose
```

Comment:
I think there has been a comment made on this somewhere, but I can't find it.

We shouldn't have dependencies and our app proxied by the same gateway. This would allow callers to call directly through to nexpose with our service's credentials.

Reply:
For an actual implementation of this, I absolutely agree. My understanding though was that this api.yaml though was more of an example, so I wasn't following that. I'll split this.

Reply:
Yeah that's fair. I think i'll mark my comment as a suggestion rather than a task. Yes, this api.yaml won't actually be used, but the examples we set, ideally, would serve as a close to real world example that someone could actually lift and build on.

Reply:
I agree with setting a good real world example, so I split them.